Makefile

```diff
@@ -0,0 +1,81 @@
+#
+# Makefile for gemplotting package
+#
+
+BUILD_HOME := $(shell dirname `pwd`)
+
+Project      := gem-plotting-tools
+ShortProject := gemplotting
+Namespace    := gempython
+Package      := gem-plotting-tools
+ShortPackage := gemplotting
+LongPackage  := gemplotting
+PackageName  := $(Namespace)_$(ShortPackage)
+PackageDir   := pkg/$(Namespace)/$(ShortPackage)
+ScriptDir    := pkg/$(Namespace)/scripts
+
+# Explicitly define the modules that are being exported (for PEP420 compliance)
+PythonModules = ["$(Namespace).$(ShortPackage)", \
+                 "$(Namespace).$(ShortPackage).utils", \
+                 "$(Namespace).$(ShortPackage).fitting", \
+                 "$(Namespace).$(ShortPackage).macros", \
+                 "$(Namespace).$(ShortPackage).mapping" \
+]
+$(info PythonModules=${PythonModules})
+
+GEMPLOTTING_VER_MAJOR=1
```

Comment:
Is the user expected to upgrade the version numbers manually?

Reply:
user, no, maintainer, yes
Workflow:
* develop feature - `developer` domain
* PR to "staging" area - `developer` domain
* release to "releasing" area - `maintainer` domain
* make a new tagged release - `maintainer` domain

Reply:
Let's wait for @jsturdy to respond.  If this is not satisfying @mexanick I would suggest a follow-up issue/PR in which the versions are set by a query of the git repo's available releases and then just adding "1" to the appropriate field.